pkg/proto/remoteactionrouter/remote_action_router.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package buildbarn.remoteactionrouter;
+
+import "build/bazel/remote/execution/v2/remote_execution.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/buildbarn/bb-remote-execution/pkg/proto/remoteactionrouter";
+
+message RouteActionRequest {
+  // The REv2 instance name prefix to operate against.
+  string instance_name_prefix = 1;
```

Comment:
s/instance_name_prefix/instance_name/, right?

This contains the literal instance name that was sent by the client. Instance name *prefixes* are only used by workers.